query/rds/rds_cluster_and_database_snapshot_encrypted_at_rest.sql

```diff
@@ -0,0 +1,37 @@
+(
+select
+  -- Required Columns
+  arn as resource,
+  case
+    when storage_encrypted then 'ok'
+    else 'alarm'
+  end as status,
+  case
+    when storage_encrypted then title || ' db cluster snapshot encrypted.'
```

Comment:
```suggestion
    when storage_encrypted then title || ' encrypted at rest.'
```

Please make similar changes for the other reasons below